### PR TITLE
Shuffle: Update to use Parse::List Role

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
###### Description of changes

Updates the Shuffle Instant Answer to use the Parse::List Role.
###### People to notify

@mintsoft 

---

Instant Answer Page: https://duck.co/ia/view/shuffle

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @GuiltyDolphin

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/Shuffle.t [hunk 1]
```diff
@@ -62,16 +62,8 @@ ddg_goodie_test(
...
-    'shuffle [1, 2]' => build_test([1, 2]),
-    'shuffle (1, 2)' => build_test([1, 2]),
-    'shuffle {1, 2}' => build_test([1, 2]),
-    # In words
-    'shuffle 1 and 2 and 3'  => build_test([1, 2, 3]),
-    'shuffle 1, 2, and 3'    => build_test([1, 2, 3]),
-    'shuffle 1 and 2, and 3' => build_test([1, 2, 3]),
     # Ranges
-    'shuffle (1..3)'   => build_test([1, 2, 3]),
+    'shuffle [1..3]'   => build_test([1, 2, 3]),
```
Comment (mintsoft) [inline t/Shuffle.t]: I presume all these bracket combos are covered by the role?

Comment (GuiltyDolphin) [inline t/Shuffle.t]: Yeah.

**Comment by moollaza** [comment]
Ping! What's the status of this? Ready for review + merge?

**Comment by mintsoft** [comment]
@moollaza This is old, but I don't see any reason not to merge this. Seems solid, the role is already in etc. Any objections?

**Pull request merged**